src/Components/Blazor/WebAssembly.Authentication/src/RemoteAuthenticatorView.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// An <see cref="RemoteAuthenticatorViewCore{TAuthenticationState}"/> that uses <see cref="RemoteAuthenticationState"/> as the
+    /// state to be persisted across authentication operations.
+    /// </summary>
+    public class RemoteAuthenticatorView : RemoteAuthenticatorViewCore<RemoteAuthenticationState>
```

Comment:
This is looking really good now.

One thing I'd like to check to complete my understanding: do we *already* have use cases for needing to inherit directly from `RemoteAuthenticatorViewCore<T>` elsewhere (e.g., for msal or something)? If we do then great, this all makes sense. If we don't yet have reasons to do that and the base class only exists for future extensibility, we should consider making the base class internal for now.